Foundations/Components/ProfileCard.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  ProfileCard.swift
+//  Foundations
+//
+//  Created by José Danilo Silva do Carmo on 28/03/23.
+//
+
+import SwiftUI
+
+struct ProfileCard<Content: View>: View {
+    @State public var title: String;
+    @ViewBuilder public var content: Content;
+
+    var body: some View {
+        VStack {
+            Text(title)
+                .foregroundColor(
+                    Color("black-color"))
+                .font(.custom("Inter-Bold",
+                              size: 17))
+                .fontWeight(.bold)
+            Spacer()
```

Comment:
Not a comment, but rather a question - is there any spacer that also outputs a straight line?

Reply:
https://developer.apple.com/documentation/swiftui/divider